index.d.ts

```diff
@@ -28,7 +28,7 @@ declare namespace OttomanJS {
 
   type ValidatorFunction = (value: any) => void
   interface SchemaField {
-    type: 'string' | 'number' | 'integer' | 'boolean' | 'Date' | 'Mixed' | ModelReference,
+    type: 'string' | 'number' | 'integer' | 'boolean' | 'Date' | 'mixed' | ModelReference,
```

Comment:
I see this "mixed" change which is basically the opposite that Will noticed last night. Why from 'Mixed' to 'mixed'?

Reply:
I did not change that, weird :/ maybe by mistake somehow